Rubberduck.Parsing/Annotations/DescriptionAnnotation.cs

```diff
@@ -8,12 +8,12 @@ namespace Rubberduck.Parsing.Annotations
     /// <summary>
     /// Used for specifying a member's <c>VB_Description</c> attribute.
     /// </summary>
-    public sealed class DescriptionAnnotation : AnnotationBase, IAttributeAnnotation
+    public sealed class DescriptionAnnotation : AttributeAnnotationBase
     {
         public DescriptionAnnotation(QualifiedSelection qualifiedSelection, VBAParser.AnnotationContext context, IEnumerable<string> parameters)
-            : base(AnnotationType.Description, qualifiedSelection, context)
+            : base(AnnotationType.Description, qualifiedSelection, context, parameters.Take(1).ToList())
```

Comment:
I'd be more comfortable with a null check here: `parameters?.Take(1).ToList() ?? Enumerable.Empty<string>()`. This is a potential NRE in the ctor.

Reply:
Thanks for catching this. Will amend.